builder/views.py

```diff
@@ -163,14 +163,12 @@ def update(request, draft):
 @require_http_methods(["POST"])
 @load_draft
 def new_search(request, draft):
-    if request.POST["field"] == "term":
-        term = request.POST["search"]
-        code = None
-    elif request.POST["field"] == "code":
+    term = request.POST["search"].strip()
+    if term.startswith("code:"):
+        code = term[5:].strip()
```

Comment:
Would `str.removeprefix("code:")` be of use here?

Reply:
TIWR (today I was reminded...)

Reply:
Except we're not running on 3.9 :(

Reply:
One to come back to later then 🙂